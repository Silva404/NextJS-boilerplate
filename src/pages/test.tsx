import { render, screen } from '@testing-library/react'
import Main from '.'

describe('main', () => {
  it('should render the heading', () => {
    const { container } = render(<Main />)

    expect(
      screen.getByRole('heading', { name: /primeiro teste/i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })

  it('should render colors correctly', () => {
    const { container } = render(<Main />)

    expect(container.firstChild).toHaveStyle({ background: '#000' })
  })
})
