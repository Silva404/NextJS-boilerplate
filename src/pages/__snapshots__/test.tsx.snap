// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`main should render the heading 1`] = `
.c0 {
  background: #000;
}

<div
  class="c0"
>
  <h1>
    Primeiro teste
  </h1>
</div>
`
